core/container/pool/pool.odin

```diff
@@ -0,0 +1,96 @@
+package container_pool
+
+import "core:fmt"
+
+// Single bit to mark a slot as active
+PoolFlag :: bit_set[0..<64;u64]
+
+// Dynamically resizable generic data pool
+Pool :: struct($type: typeid) {
+    data : [dynamic]type,
```

Comment:
The core library style does not use a space before `:` in a declaration, unless it is directly followed by `=`.